### PR TITLE
Minor suggestions

### PR DESCRIPTION
**Pull request by bsipocz**
We need to use the new paper reference, otherwise everything else is a minor suggestion.

### CODE REVIEW

File: main.tex [hunk 1]
```diff
@@ -84,42 +84,39 @@ \section{Introduction}
...
 
 In addition to observatories and telescopes, individual surveys often share
 their full data sets.  In some cases, these data sets are shared via the
 observatory that acquired them, for example, the all-sky data acquired with
 Planck, WMAP, and COBE\@.  Other surveys, particularly ground-based surveys,
-serve their own data.  Examples include SDSS, 2MASS, UKIDSS, and likely many
-more in the near future.
+serve their own data.  Examples include SDSS, 2MASS, UKIDSS, VSA.
+
+%TODO: mention data collector sites, too, nasa\_exoplanet\_archive, simbad,
```
Comment (bsipocz) [inline main.tex]: In this section it wasn't clear whether we mean to talk about archives in general or only the ones we have a module for. The ones mentioned in the comment don't really fit in any of the above categories, but I think are rather important.

File: main.tex [hunk 2]
```diff
@@ -84,42 +84,39 @@ \section{Introduction}
...
+custom websites, too.  These services do not follow any particular standard and can
 be widely varied in the type and amount of data shared.  Sometimes these data
 are shared via the archive systems (e.g., IRSA at IPAC hosts many individual
 survey data sets), while others use their own web hosting systems \citep[e.g.,
 MAGPIS;][]{Helfand2006}.
 
-Finally, there are other data types relevant to astronomy that are not served
-by the typical astronomical databases.  Examples include databases of molecular
-and atomic properties, such as those provided by
-Splatalogue\footnote{\url{http://www.splatalogue.net/}} and the NIST Atomic
```
Comment (bsipocz) [inline main.tex:113]: I don't think we need to do footnotes here, if yes, why there is no footnote for e.g. the mast archive or 2MASS, or any of the others above.

File: main.tex [hunk 3]
```diff
@@ -331,10 +328,9 @@ \section{Development history and status}
 80NSSC18K0987. Further Solar System-related services are planned to be added to
 \astroquery through this support.
 
-
 \astroquery has received support from the Google Summer of Code
 program, with two students (co-authors Madhura Parikh and Simon Liedtke)
-from 2013--2017.
+from 2013--2014.
```
Comment (bsipocz) [inline main.tex:333]: I don't have strong preference either way, but if we go with the original one, then update the date for 2018

File: main.tex [hunk 4]
```diff
@@ -40,9 +40,9 @@
 
 \begin{abstract}
 \astroquery is a collection of tools for requesting data from databases hosted
-on remote servers with interfaces exposed on the internet, including those with
+on remote servers with interfaces exposed on the internet particular focus to those with
```
Comment (keflavich) [inline main.tex]: There is at least one missing word here; I'm inclined to go with the original

File: main.tex [hunk 5]
```diff
@@ -40,9 +40,9 @@
...
 web pages but without formal application program interfaces (APIs).  These
-tools are based on the Python
+tools are build on the Python
```
Comment (keflavich) [inline main.tex]: built

File: main.tex [hunk 6]
```diff
@@ -84,42 +84,39 @@ \section{Introduction}
...
+primary means for serving data to users when the data are taken in queue
+mode, i.e., when the data are taken while the observer is not on-site.
 
 In addition to observatories and telescopes, individual surveys often share
 their full data sets.  In some cases, these data sets are shared via the
 observatory that acquired them, for example, the all-sky data acquired with
 Planck, WMAP, and COBE\@.  Other surveys, particularly ground-based surveys,
-serve their own data.  Examples include SDSS, 2MASS, UKIDSS, and likely many
-more in the near future.
+serve their own data.  Examples include SDSS, 2MASS, UKIDSS, VSA.
```
Comment (keflavich) [inline main.tex]: need an 'and' in this list

File: main.tex [hunk 7]
```diff
@@ -84,42 +84,39 @@ \section{Introduction}
...
+
+%TODO: mention data collector sites, too, nasa\_exoplanet\_archive, simbad,
+%skyview, OAC, etc.
 
-Individual teams and small groups will often share their data via their own
-custom websites.  These services do not follow any particular standard and can
+Individual teams and small groups often share their data via their own
+custom websites, too.  These services do not follow any particular standard and can
```
Comment (keflavich) [inline main.tex]: the `, too` is not needed.  I make an effort to remove extraneous 'also's, 'too's, and 'additionally's from my work because I tend to write them too often, and they don't add anything.

File: main.tex [hunk 8]
```diff
@@ -84,42 +84,39 @@ \section{Introduction}
...
-Solar System ephemerides\footnote{e.g.,
-\url{https://ssd.jpl.nasa.gov/?ephemerides}}.  
-
-\astroquery arose from a desire to access these databases from the command line
+Finally, there are other data types relevant to astronomy that are not
+served by the typical astronomical databases.  Examples include databases of
+molecular and atomic properties, such as those provided by Splatalogue and
+the NIST Atomic Spectra Database, or services that are computationally
+intensive or require constant updates, like Solar System ephemerides
+provided by services like JPL HORIZONS, or the Minor Planet Center.
```
Comment (keflavich) [inline main.tex:115]: Why did you remove the footnotes?

File: main.tex [hunk 9]
```diff
@@ -142,27 +139,28 @@ \section{The Software}
 a common framework that uses the Python \package{requests} package to perform
 HTTP requests to communicate with web services.
 
-For new development, there is a \texttt{template}  that lays out the basic
-framework of any new module.  All modules have a single core
+For new service development, there is a \texttt{template module} that lays
```
Comment (keflavich) [inline main.tex]: `module` should not be within the `\texttt{}` segment

File: main.tex [hunk 10]
```diff
@@ -142,27 +139,28 @@ \section{The Software}
...
+An example using the SIMBAD interface is shown below:\footnote{see
+\url{http://astroquery.readthedocs.io/en/latest/simbad/simbad.html}}
 \begin{lstlisting}[caption=Query SIMBAD for a region around M81]
 from astroquery.simbad import Simbad
 result_table = Simbad.query_region("m81")
 \end{lstlisting}
 In this example, \texttt{Simbad} is an instance of the
 \texttt{astroquery.simbad.SimbadClass} class.
-The returned result, stored in the variable \texttt{result\_table},
-is an \astropypkg table\footnote{\url{http://docs.astropy.org/en/stable/table/}}.
```
Comment (keflavich) [inline main.tex:161]: Why is this sentence removed?  It's important to specify what is returned

File: main.tex (hunk 9 above)
Comment (keflavich) [reply main.tex]: to clarify, `template` is the name of the module as it appears in the repository; it's technically importable.  "module" is just an English description of that object

File: main.tex (hunk 10 above)
Comment (keflavich) [reply main.tex:161]: Oh, I see, you just moved it up.  That's OK

File: main.tex [hunk 11]
```diff
@@ -142,27 +139,28 @@ \section{The Software}
...
 
-While there is a common suggested API described in the \texttt{template} module,
+While there is a common suggested API described in the \texttt{template module},
 individual packages are not \emph{required} to support this API because, for
-some, it is not possible.  For example, the atomic and molecular databases refer
+some, it is not practical or even possible.  For example, the atomic and molecular databases refer
```
Comment (keflavich) [inline main.tex]: leave this as just 'possible'.

**Review by keflavich — CHANGES_REQUESTED** [review]
There are a few fixes needed before I merge this.  

Once you're done, let me know if you're happy with submission.  I have the document ready to go into ApJ immediately.

File: main.tex (hunk 8 above)
Comment (bsipocz) [reply main.tex:115]: ohh, sorry, it seems I never sent my comments. For this one I wrote: I don't think we need to do footnotes here, if yes, why there is no footnote for e.g. the mast archive or 2MASS, or any of the others above.

File: main.tex (hunk 9 above)
Comment (bsipocz) [reply main.tex]: well, it's called `template_module` isn't it, while the class is `Template`

File: main.tex (hunk 10 above)
Comment (bsipocz) [reply main.tex:161]: yes, I think it belong to the description above as it hasn't said anything about the preferred return objects.
